shake.project

```diff
@@ -0,0 +1,2 @@
+packages:
```

Comment:
add an additional cabal.project in the `install/` such that we can compile the executable from hand if we want to

Reply:
done